esmvaltool/recipes/recipe_climwip_brunner20esd.yml

```
@@ -33,7 +33,7 @@ datasets: &model_data
     - {dataset: CanESM5, project: CMIP6, grid: gn, exp: [historical, ssp585], ensemble: r(1:25)i1p1f1}
     - {dataset: CanESM5, project: CMIP6, grid: gn, exp: [historical, ssp585], ensemble: r(1:25)i1p2f1}
     - {dataset: CESM2-WACCM, project: CMIP6, grid: gn, exp: [historical, ssp585], ensemble: r1i1p1f1}
-    - {dataset: CESM2, project: CMIP6, grid: gn, exp: [historical, ssp585], ensemble: r(1:2)i1p1f1}
+    #- {dataset: CESM2, project: CMIP6, grid: gn, exp: [historical, ssp585], ensemble: r(1:2)i1p1f1} # missing ssp585 data
```

```suggestion
    #- {dataset: CESM2, project: CMIP6, grid: gn, exp: [historical, ssp585], ensemble: r(1:2)i1p1f1} # ssp585 data missing on ESGF
```

or alternatively, link to this PR as it quite clearly documents why we have removed it.